packages/search-ui/src/SearchDriver.ts

```diff
@@ -277,13 +279,38 @@ class SearchDriver {
       ...searchParameters
     };
 
+    if (initialResponseState) {
+      const { current, resultsPerPage } = this.state;
+
+      const { totalResults } = initialResponseState;
+
+      // Results paging start & end
+      const start = totalResults === 0 ? 0 : (current - 1) * resultsPerPage + 1;
+
+      const end =
+        totalResults < start + resultsPerPage
+          ? totalResults
+          : start + resultsPerPage - 1;
+
+      this.state = {
+        ...this.state,
+        ...{
+          pagingStart: start,
+          pagingEnd: end,
+          wasSearched: true,
+          ...initialResponseState
+        }
+      };
+    }
+
     // We'll trigger an initial search if initial parameters contain
     // a search term or filters, or if alwaysSearchOnInitialLoad is set.
     // Otherwise, we'll just save their selections in state as initial values.
```

Comment:
Update this comment with the extra clause below?

Reply:
Done